Library/QuickPulseSender.ts

```diff
@@ -114,7 +115,8 @@ class QuickPulseSender {
         const req = https.request(options, (res: http.IncomingMessage) => {
             if (res.statusCode == 200) {
                 const shouldPOSTData = res.headers[QuickPulseConfig.subscribed] === "true";
-                const redirectHeader = res.headers[QuickPulseConfig.endpointRedirect] ? res.headers[QuickPulseConfig.endpointRedirect].toString() : null;
+                const redirectHeader = res.headers[QuickPulseConfig.endpointRedirect] ? new url.URL(res.headers[QuickPulseConfig.endpointRedirect].toString()).host : null;
```

Comment:
new URL() could throw an exception, please try/catch and log error

Reply:
Fixed :)